clients/es-integration/src/integration-test/resources/bootstrap.yml

```diff
@@ -0,0 +1,20 @@
+spring:
+  rabbitmq:
+    host: ${RABBIT_HOST:localhost}
+    username: ${RABBIT_USERNAME:guest}
+    password: ${RABBIT_PASSWORD:guest}
+  data:
+    elasticsearch:
+      cluster-nodes: ${ELASTICSEARCH_CLUSTER_NODES:localhost:9300}
+      cluster-name: ${ELASTICSEARCH_CLUSTER_NAME:tweets-cluster}
+  cloud:
```

Comment:
cloud in integration tests?!

Reply:
This configuration disables config server and service discovery